### ISSUE TITLE
I need help

### ISSUE THREAD

**Issue by gaoruiqing187** [ProxymanApp/Proxyman#1983]
## Description
I'm trying to install the certificate on my Android phone, but the http://proxy.man/ssl link won't open. I have all the proxies set up as requested

**Comment by NghiaTranUIT** [comment]
@gaoruiqing187 this website is not a real website, it's available when you set up the Proxy from your Android Phone to Proxyman app properly.

Can you read the [Troubleshooting](https://docs.proxyman.io/troubleshooting/my-ios-devices-couldnt-connect-to-proxyman-via-proxy) and try to find solution?
Maybe:
- You set incorrect Proxy 
- Firewall on your Macbook
- You use VPN on your Android device.
- ...

**Comment by gaoruiqing187** [comment]
@NghiaTranUIT I have read the troubleshooting instructions and have tried as required, but it still doesn't work. I can't open the URL to download the certificate, and now I can't download it on my iPhone either

**Comment by NghiaTranUIT** [comment]
@gaoruiqing187 May I ask: If you set HTTP Proxy to Proxyman from your iPhone/Android device, can you visit www.google.com ?

If yes, proxy.man/ssl will work.

May I ask what Proxyman version you're using? macOS or Windows/Linux?

**Comment by gaoruiqing187** [comment]
I can't visit google, I'm using version 4.14.0 (49500)， MacOS

**Comment by NghiaTranUIT** [comment]
yeah, so it means the HTTP Proxy doesn't work, not `proxy.man/ssl`

Can you verify:
- Turn off the Firewall on your Macbook
- Your Mac & Android are connected to the same Wifi
- Turn off any VPN app on your Android
- Try to set the IP + Port again in the Certificate menu -> Install for Android -> Physical Device
- If it's iOS 16, 17, you should check this: https://docs.proxyman.io/troubleshooting/ios-16-devices-issues
